SegmentEditorSurfaceCut/SegmentEditorSurfaceCutLib/SegmentEditorEffect.py

```diff
@@ -165,6 +175,13 @@ def updateGUIFromMRML(self):
     if operationName != "":
       operationButton = list(self.buttonToOperationNameMap.keys())[list(self.buttonToOperationNameMap.values()).index(operationName)]
       operationButton.setChecked(True)
+    
+    if self.scriptedEffect.parameter("SmoothModel") == 'True':

```

Comment:
This is correct, but for consistency with how other effects store Boolean values, please use 0/1 instead. See example here:

https://github.com/Slicer/Slicer/blob/62db2030cba19974d8e0cd556c084a9e00049f4f/Modules/Loadable/Segmentations/EditorEffects/Python/SegmentEditorLogicalEffect.py#L141-L144